app/_gateway_entities/consumer.md

```diff
@@ -78,7 +78,7 @@ schema:
 
 ## What is a Consumer?
 
-{{ page.description }} Consumers can be applications, services, or users who interact with your APIs.
+{{ page.description | liquify }} Consumers can be applications, services, or users who interact with your APIs.
```

Comment:
```suggestion
A Consumer typically refers to an entity that consumes or uses the APIs managed by {{site.base_gateway}}. Consumers can be applications, services, or users who interact with your APIs.
```
Fabian recommended just putting the description directly in the intro paragraph on one of my other PRs.